### ISSUE TITLE
Evaluate Guides static site generator candidates

### ISSUE THREAD

**Issue by beechnut** [18F/TLC-crew#169]
### Evaluate Guides static site generator candidates

18F Engineering is leading the process of choosing a static site generator for our Guides and Methods. ([See announcement.](https://gsa-tts.slack.com/archives/CBUUX8FKQ/p1680885062079069))

In order to choose a static site generator, we will evaluate several options for generators against our decision criteria.

The decision criteria are captured in [the decision criteria spreadsheet.](https://docs.google.com/spreadsheets/d/1EDDbnT4nHPj4MlH9CcLoZxO2sBVzT4YI3UEzbewxZs8/edit#gid=0)

We are requesting help in doing **research** to determine how well each option meets the decision criteria.

This will move 18F towards fulfilling its vision for the Guides and Methods, and will unblock related work for the TLC team.

### Point of contact on this issue
@beechnut (owner in [MOCHA](https://www.managementcenter.org/resources/assigning-responsibilities/))
@adunkman (manager, backup)

**Billable?**

- [ ] Yes
- [x] No
If yes, tock code:

**Skills needed**

- [ ] Any human
- [ ] Design
- [ ] Content
- [x] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

1 engineer

**Timeline**
Does this need to happen in the next two weeks?

- [x] Yes - research could start today
- [ ] No

How much time do you anticipate this work taking?:
4-8 hours (1 hour per framework, plus alignment/debrief time)

### Acceptance Criteria

- [x] The decision criteria spreadsheet is filled out for all generator options

**Comment by alexbielen** [comment]
Increment planning risks and changes: 

- Are there opportunities to add a designer to the evaluation?
- Is this .... already done? Checking that this makes sense to take on still.
- Is this more about making a decision now?
- As a new person, I'm noticing that I don't really have a lot of context on what needs to be done. There is broader organizational context that I feel like I might need. Should I expect that to be in these tickets?

**Comment by beechnut** [comment]
@alexbielen I'm not sure if these are all questions for me, but I can say that this is not done. While the spreadsheet has some content, we still need help investigating a selection of high-risk criteria for greater certainty.

**Comment by annepetersen** [comment]
Existing prototypes that could be helpful, for context: https://github.com/18F/guides-and-methods-hugo/
https://github.com/18F/sd-11ty-uswds-demo or https://github.com/18F/federalist-uswds-11ty

**Comment by julialeague** [comment]
@beechnut might weigh in with further detail, but the plan right now is to have a prototype of the three top SSGs, which are: Astro, Eleventy, and Hugo. Thanks @annepetersen for the context!

**Comment by alexbielen** [comment]
Great! On Monday I'll create the prototype issues each of which appears to have some individualized context. I'm going to set up some time with @beechnut as well to make sure we're on the same (statically generated) page 😄

**Comment by alexbielen** [comment]
Closing this as done since the acceptance criteria are met.
